packages/labs/task/src/task.ts

```diff
@@ -194,7 +196,6 @@ export class Task<T extends [...unknown[]] = any, R = any> {
     }
     this.status = TaskStatus.PENDING;
     this._error = undefined;
-    this._value = undefined;
```

Comment:
This is the significant change that I want to make sure we're documenting and testing.

Reply:
First I think we need to decide that this is the behavior we want. Is it required to add an initial value? Should it be controlled with an option? Let's chat.